resources/assets/coffee/react/_components/beatmapset-mapping.coffee

```diff
@@ -20,7 +20,7 @@ el = React.createElement
 
 bn = 'beatmapset-mapping'
 
-@BeatmapsetMapping = ({user, beatmapset}) ->
+@BeatmapsetMapping = ({user, beatmapset, beatmap}) ->
```

Comment:
what I'm saying is @peppy said this particular react component should show the `beatmapset` approved status, not `beatmap`.